docs/security/host-certs/digicert.md

```diff
@@ -0,0 +1,119 @@
+DigiCert IGTF Host Certificates
+===============================
+
+!!! note
+    This document describes how to get **host** certificates.
+    For instructions on how to get **user** certificates, see the [User Certificates document](/security/user-certs).
+
+This document describes how to purchase individual IGTF-accredited host certificates from DigiCert[DigiCert](https://www.digicert.com/).
+Before purchasing individual certificates, consider the following alternatives:
```

Comment:
GitHub still doesn't support multi-line suggestions but the syntax of the following list doesn't match this sentence. Try this:

```
1. Request a [Let's Encrypt certificate](/security/host-certs/lets-encrypt.md)
   if you don't support any VOs that require IGTF-accredited certificates (e.g. `ATLAS` or `CMS`).
1. Request an [InCommon certificate](/security/host-certs/incommon.md)
   if your institution has an InCommon subscription.
```